content/en/docs/project-user-guide/application-workloads/deployments.md

```diff
@@ -7,4 +7,291 @@ description: 'Kubernetes Deployments'
+    - **Success Threshold**: is `successThreshold`. Minimum consecutive successes for the probe to be considered successful after having failed. It defaults to 1 and must be 1 for liveness and startup. The minimum value is 1.
+    - **Failure Threshold**: is `failureThreshold`. Minimum consecutive failures for the probe to be considered failed after having succeeded. It defaults to 3 and the minimum value is 1.
+
+  - **ExecAction**: Execute the specified command in the container. If the return code is 0 when the command exits, the diagnosis is considered successful. The supported parameters include:
+    - **Command**: is `exec.command`. A detection command used to detect the health of the container.
+    - **Initial Delays**: is `initialDelaySeconds`. The number of seconds after the container has started before liveness probes are initiated, the default is 0 seconds.
+    - **Period Seconds**: is `periodSeconds`.
+      Probe frequency (in seconds), which defaults to 10 seconds. The minimum value is 1.
+    - **Timeouts**: is `timeoutSeconds`. Number of seconds after which the probe times out. It defaults to 1 second and the minimum value is 1.
+    - **Success Threshold**: is `successThreshold`. Minimum consecutive successes for the probe to be considered successful after having failed. It defaults to 1 and must be 1 for liveness and startup. The minimum value is 1.
+    - **Failure Threshold**: is `failureThreshold`. Minimum consecutive failures for the probe to be considered failed after having succeeded. It defaults to 3 and the minimum value is 1.
+
+   For more information about health checks, please visit [Container Probes](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
+
+- **Start Command**
+
+  By default, the container runs the default image command.
+
+  - Run Command is the `command` of containers.
+  - Parameters are the `args` of containers.
+
+  For more information about the command, please visit [Container Command](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/).
+
+- **Environment Variables**
+
+  Supports configuring environment variables for Pod in the form of key-value pairs.
+
+  - name：Set the name of the environment variable.
+  - value：Set the value of the variable reference.
+  - type：Set the type of environment variables, support customization, configuration items, keys, and variable/variable references.
+
+  For more information about the command, please visit [Pod variable](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/?spm=a2c4g.11186623.2.20.16704b3e9qHXPb).
+
+- **Container Security Context**
+
+  A security context defines privilege and access control settings for a Pod or Container. [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
```

Comment:
[Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
->
For more information about the security context, see [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).